libraries/chain/custom_authority_object.cpp

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2015 Cryptonomex, Inc., and contributors.
```

Comment:
Please replace this by
`Copyright (c) 2015 BitShares Core Team, and contributors.`

Reply:
done